src/main/resources/META-INF/resources/jsp/application/include/app-detail-elements/vacation_overview_js.jsp

```diff
@@ -50,100 +50,54 @@
 											.forEach(
 													function(currentDay, index,
 															array) {
-														if (response.response.absences
-																.find(
-																		function(
-																				currentValue,
-																				index,
-																				array) {
-																			if (this.toString() == currentValue.date
-																					&& currentValue.status === "WAITING"
-																					&& currentValue.type === "VACATION"
-																					&& currentValue.dayLength === 1) {
-																				return "test";
-																			}
-																		},
-																		currentDay.dayText)) {
-															currentDay.cssClass = 'vacationOverview-day-personal-holiday-status-WAITING vactionOverview-day-item ';
+														var currentAbsence = response.response.absences
```

Comment:
Hier ist der Diff etwas länger geworden, weil noch ein kleines bisschen Refactoring hereingerutscht ist, welches zwar nicht zwingend notwendig ist, ich aber für sehr angebracht halte.

Die alte Version benutzt `Array.find` wiederholt um den aktuellen Tag im Datensatz zu finden und gleichzeitig auf verschiedene Bedingung zu testen. 
Die neue Version zieht `currentAbsence` nur einmal (als Zwischenergebnis) heraus, und testet dann dieses. Das Array muss nicht mehrmals durchlaufen werden.

Immerhin befindet sich diese Schleife in einer sehr oft aufgerufenen Method (für jeden angezeigten Tag X jeden angezeigten Mitarbeiter). Ausserdem ist der Code so kompakter.

Die einzige funktionelle Änderung ist, dass allen gesetzten CSS-Klassen das neue `categoryCss` angehängt wird.